root/static/scripts/common/components/Autocomplete2/reducer.js

```diff
@@ -201,7 +201,7 @@ export function generateStatusMessage<+T: EntityItem>(
   return '';
 }
 
-function filterStaticItems<+T: EntityItem>(
+export function filterStaticItems<+T: EntityItem>(
```

Comment:
You export this, but you don't seem to be using it elsewhere?

Reply:
Oh, true, though I do import it somewhere in my relationship editor branch, so probably fine to leave it.